projects/storefrontlib/src/cms-components/asm/services/asm-enabler.service.ts

```diff
@@ -71,6 +70,6 @@ export class AsmEnablerService {
    * Adds the ASM UI by using the `cx-storefront` outlet.
    */
   protected addUi(): void {
-    this.triggerService.render(TRIGGER_CALLER.ASM, AsmMainUiComponent);
+    this.triggerService.render(TRIGGER_CALLER.ASM);
```

Comment:
❤️

Reply:
`ComponentFactoryResolver` and `TriggerConfig` are no longer needed in this class